python/plugins/processing/algs/qgis/PointsToPaths.py

```diff
@@ -144,10 +144,10 @@ def processAlgorithm(self, parameters, context, feedback):
 
             point = f.geometry().constGet().clone()
             if group_field_index >= 0:
-                group = f.attributes()[group_field_index]
+                group = f[group_field_index]    # (1)
```

Comment:
what does it means # (1) or # (2) ?

Reply:
I just labeled the changes mentioned in the pull request description 1 and 2.

Reply:
Right - these will need to be removed, as they have no meaning outside of this pr discussion